R/transition_risk_profile.R

```diff
@@ -0,0 +1,128 @@
+#' Calculate the indicator "transition risk profile"
+#'
+#' Adds the risk classification to calculated transition risk scores from
+#' emission profile and sector profile indicator.
+#'
+#' @param emissions_profile Nested data frame. The output of
+#'   `profile_emissions()`.
+#' @param sector_profile Nested data frame. The output of `profile_sector()`.
+#' @param pivot_wider Logical. Pivot the output at company level to a wide
+#'   format?
+#' @param emissions_profile_products A dataframe
+#' @param all_uuids_scenario_sectors A dataframe
+#' @param scenarios A dataframe
+#'
+#' @return A data frame with the column `companies_id`, and the nested
+#'   columns`product` and `company` holding the outputs at product and company
+#'   level.
+#' @export
+#'
+#' @examples
+#' library(readr, warn.conflicts = FALSE)
+#' library(dplyr, warn.conflicts = FALSE)
+#' library(tiltToyData, warn.conflicts = FALSE)
+#' library(tiltTransitionRisk, warn.conflicts = FALSE)
+#'
+#' set.seed(123)
+#' restore <- options(list(
+#'   readr.show_col_types = FALSE,
+#'   tiltIndicatorAfter.output_co2_footprint = TRUE
+#' ))
+#'
+#' toy_emissions_profile_products_ecoinvent <- read_csv(toy_emissions_profile_products_ecoinvent())
+#' toy_emissions_profile_any_companies <- read_csv(toy_emissions_profile_any_companies())
+#' toy_sector_profile_any_scenarios <- read_csv(toy_sector_profile_any_scenarios())
```

Comment:
I think it's clearer to name each of these variables using the name of the argument they will be pass to. 

For example:

```suggestion
#' scenarios <- read_csv(toy_sector_profile_any_scenarios())
```

Note you can  reuse `scenarios` (see below).